proposals/notifications.md

```diff
@@ -0,0 +1,169 @@
+# Solid inboxes and notifications
+
+Solid inboxes are repositories for incoming social interactions and data. A
+Solid inbox is an [LDP Container](https://www.w3.org/TR/ldp/#ldpc) for Solid
+notifications. A resource can have one or more inboxes in
+order to facilitate different social interactions. For instance, user profile
+inboxes are typically intended to receive notifications like messages from other
+profiles, or application specific activities.
+
+By default, Solid servers should create an inbox for new profiles, and have the
```

Comment:
I would use `accounts` instead of `profiles`. Creating the inbox only takes place if the account also creates an identity (i.e. a WebID profile).